windows-driver-docs-pr/install/inf-addservice-directive.md

```diff
@@ -244,6 +245,14 @@ For information about security descriptor strings, see [Security Descriptor Defi
 
 For more information about how to specify security descriptors, see [Creating Secure Device Installations](creating-secure-device-installations.md).
 
+**RequiredPrivileges**=*privilege-name*
+> [!NOTE]
+> This value can only be used for *Win32 Services* and is only available with Windows 10 Version 2004 and above.
```

Comment:
This is available on Windows 7 and above, not Windows 10 Version 2004 and above.